### PR TITLE
repair config path for ember generators

### PR DESCRIPTION
**Pull request by stakoov**
When use custom path for ember app pop up errors for path config.

### CODE REVIEW

**Comment by rwjblue** [comment]
Looks good. Could you add an entry to the changelog?

**Comment by bcardarella** [comment]
This is failing right now, the test needs to be fixed

**Comment by stakoov** [comment]
add changelog + fixed test

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ## 0.5.0
-
+* fixed  proper paths setting "config.ember.appkit.paths.app" and "config.ember.appkit.paths.config" to "config.ember.paths.app" and "config.ember.paths.config"
```
Comment (bcardarella) [inline CHANGELOG.md]: this needs to be added at the bottom of the release version notes

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,4 @@
 ## 0.5.0
-
```
Comment (bcardarella) [inline CHANGELOG.md:2]: Can you restore this new line?

**Comment by bcardarella** [comment]
This will also need to be rebased

**Comment by stakoov** [comment]
All must be ok now just had some weird error while rebase

**Comment by bcardarella** [comment]
Great, thank you!

**Pull request merged**